tests/request_helper_test.py

```diff
@@ -129,6 +130,53 @@ def test_create_zipkin_attr_runs_custom_is_tracing_if_present(dummy_request):
     is_tracing.assert_called_once_with(dummy_request)
 
 
+@pytest.mark.parametrize('is_sampled,emit_headers,headers_are_set', [
+    (True, True, True),
+    (True, False, True),
+    (False, True, True),
+    (False, False, False),
+    ])
```

Comment:
I knew this looked familiar, it is a truth table for OR. I think it's fine to just say `headers_are_set = is_sampled or emit_headers`.